.asf.yaml

```diff
@@ -0,0 +1,36 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+notifications:
+  commits:      commits@cassandra.apache.org
+  issues:       commits@cassandra.apache.org
+  pullrequests: pr@cassandra.apache.org
+
+github:
+  description: "GoCQL Driver for Apache Cassandra®"
+  homepage: https://cassandra.apache.org/
```

Comment:
What about https://gocql.github.io/ ? That page was not updated in 9 years.

Should we make it redirect to some subpage of `cassandra.apache.org` or the Github project page?

cc @Zariel

Reply:
That requires the donation of https://github.com/gocql/gocql.github.io

Let's do that in a separate ticket.  ()

We haven't heard from @tux21b , [Collin Glass](https://github.com/gocql/gocql.github.io/commit/383b5c798d6d2c71cc933f309567b4754d105057), or @danielchatfield .

I agree we should place it, along with other docs somewhere underneath cassandra.apache.org

Reply:
ping @martin-sucha @Zariel

Reply:
Sure, the web page is not blocking the merge of this PR, we can use the `https://cassandra.apache.org/` URL for now.